airflow/executors/executor_loader.py

```diff
@@ -51,7 +52,7 @@ def get_default_executor(cls) -> BaseExecutor:
         return cls._default_executor
 
     @staticmethod
-    def _get_executor(executor_name: str) -> BaseExecutor:
+    def _get_executor(executor_name: str) -> BaseExecutor:  # pylint: disable=too-many-return-statements
```

Comment:
Actually the whole `if` `elif` `else` can be easily simplified by just:

```python
if cond:
    return executor
if cond2:
   return executor2

return executor3
```

Then I think pylint will also not complain about too many return statements, because it is a common pattern.

Reply:
ah yeah :)

Reply:
Or simply executor = x in all branches and 'return executor' at the end if not empty

Reply:
@feluelle you are right that it can be simplified but still there will be a lot of return statements. I will fix it in other way

Reply:
@feluelle @potiuk let me know what you think now ;)